beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -1916,7 +1917,8 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
             };
         drop(cache_timer);
 
-        Ok(Attestation {
+        // TODO(eip7549) implement electra variant
```

Comment:
Just change this to `TODO(electra)`. Trying to standardize all `TODO` tags so that we can grep for them and ensure we didn't miss any.